patnorris/PWS
# Local Bebb Protocol

Integrates Bebb Protocol such that it can be used locally

To test:
Follow guide in NotesOnLocalDev under section Running Bebb Protocol locally to set up OIM and Bebb Protocol canisters locally

With local UI, test creating a new room
Create Neighbors for the new room
Make sure Neighbors are loaded for the room
Delete Neighbors for the room
Make sure deletions are persisted
